.changeset/config.json

```diff
@@ -6,7 +6,6 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["!(astro|create-astro|@astrojs/*|astro-scripts)"],
```

Comment:
This can be removed because the `workspaces` field only captures packages that we want to publish, otherwise changeset would error on unused ignores.